lib/Ohmbrewer_Rhizome.h

```diff
@@ -247,6 +253,17 @@ namespace Ohmbrewer {
          * @return Error or success code, according to the requirements specified by addSprout
          */
         int addRIMS(char* params);
+
+        /**
+         * Adds a sprout to the list and rebuids the index
+         * @param equipment The sprout being added
+         */
+        void saveNewSprout(Equipment* sprout);
```

Comment:
Weird... Are you sure the name was the problem? It *should* work, given that one takes a Equipment* and one takes a String: http://www.cplusplus.com/doc/tutorial/functions2/